mk/check.mk

```diff
@@ -54,7 +50,7 @@ format: fmt format/common
 
 .PHONY: kube-lint
 kube-lint:
-	$(KUBE_LINTER) lint deployments/charts/kuma
+	find $(HELM_CHARTS_DIR) -maxdepth 1 -mindepth 1 -type d -exec $(KUBE_LINTER) lint {} \;
```

Comment:
This will have ot be tweaked somewhat because we want to run `kube-lint` on `app/kumactl/cmd/install/testdata/install-control-plane*.golden.yaml` soon.